code cleanups

outsparkled opened the pull request.

Elikill58 commented:
Why change TAB length from 3-spaces to 2-spaces ? Else, there is good point into your PR

outsparkled commented:
> Why change TAB length from 3-spaces to 2-spaces ? 

the indentation seems to only consist of 4-spaces and tab characters. the ones i changed were mixing tabs and spaces, which i changed to only spaces
